mmdet/models/trainer.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Open-MMLab. All rights reserved.
```

Comment:
Where does this file come from?

Reply:
IMO, this not related to neither models nor training. Please find a better place and name for it.

Reply:
Moved to `mmdet / parallel / data_cpu.py`, like [torch/nn/parallel/data_parallel.py](https://github.com/pytorch/pytorch/blob/master/torch/nn/parallel/data_parallel.py) and [mmcv/parallel/data_parallel.py](https://github.com/open-mmlab/mmcv/blob/master/mmcv/parallel/data_parallel.py). Is it ok? I dont know better name.